### ISSUE TITLE
Corpo Theme - only retains 3 "retried chunks" despite being able to retain more than 3 just a few days ago

### ISSUE THREAD

**Issue by NintendoManiac64** [LostRuins/lite.koboldai.net#85]
Sometime very recently, it would seem that the Corpo Theme begun only retaining 3 "retried chunks" despite previously being able to retain... I don't know the limit, but I definitely was able to get to like 7 or so.

**Comment by LostRuins** [comment]
No, the "last turn history" has always been limited to the last 3 retries.
